### PR TITLE
test: edit calendar modal

### PR DESCRIPTION
**Pull request by FrancescoGottardi**

### CODE REVIEW

**Review by gnekoz — DISMISSED** [review]

File: src/test/mocks/network/msw/handle-get-folder.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+					s: 0,
+					i4ms: 13151,
+					i4next: 3799,
+					acl: {
+						grant: [
+							{
+								zid: '0e9d1df6-30df-4e1d-aff6-212908045221',
+								gt: 'usr',
+								perm: 'r',
+								d: 'gabriele.marino@zextras.com'
```
Comment (nubsthead) [inline src/test/mocks/network/msw/handle-get-folder.ts:42]: avoid sensible data, use fake ones

File: src/view/sidebar/edit-calendar-modal.test.tsx [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+import { useCalendarActions } from '../../hooks/use-calendar-actions';
+
+describe('main-calendar-modal', () => {
+	test('change color', async () => {
+		const closeFn = jest.fn();
+		const grant = {
+			zid: '0e9d1df6-30df-4e1d-aff6-212908045221',
+			gt: 'usr',
+			perm: 'r',
+			d: 'gabriele.marino@zextras.com'
```
Comment (nubsthead) [inline src/view/sidebar/edit-calendar-modal.test.tsx]: same here

File: src/view/sidebar/edit-calendar-modal.test.tsx [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+			expect(store.getState().calendars.calendars[folder.id].color.label).toBe('red');
+		});
+	});
+	test('change name', async () => {
+		const closeFn = jest.fn();
+		const grant = {
+			zid: '0e9d1df6-30df-4e1d-aff6-212908045221',
+			gt: 'usr',
+			perm: 'r',
+			d: 'gabriele.marino@zextras.com'
```
Comment (nubsthead) [inline src/view/sidebar/edit-calendar-modal.test.tsx]: same here

File: src/view/sidebar/edit-calendar-modal.test.tsx [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+			expect(newCalendar.name).toBe('Calendar 123');
+		});
+	});
+	test('edit free-busy', async () => {
+		const closeFn = jest.fn();
+		const grant = {
+			zid: '0e9d1df6-30df-4e1d-aff6-212908045221',
+			gt: 'usr',
+			perm: 'r',
+			d: 'gabriele.marino@zextras.com'
```
Comment (nubsthead) [inline src/view/sidebar/edit-calendar-modal.test.tsx]: same here

File: src/view/sidebar/edit-calendar-modal.test.tsx [hunk 5]
```diff
@@ -0,0 +1,265 @@
...
+			expect(screen.queryByTestId('MainEditModal')).not.toBeInTheDocument();
+		});
+	});
+	test('add share', async () => {
+		const closeFn = jest.fn();
+		const grant = {
+			zid: '0e9d1df6-30df-4e1d-aff6-212908045221',
+			gt: 'usr',
+			perm: 'r',
+			d: 'gabriele.marino@zextras.com'
```
Comment (nubsthead) [inline src/view/sidebar/edit-calendar-modal.test.tsx]: same here

**Review by nubsthead — DISMISSED** [review]

**Review by gnekoz — APPROVED** [review]

**Review by dhavaldodiya — APPROVED** [review]

**Pull request merged**